### PR TITLE
Mattiacmhpc utilities

### PR DESCRIPTION
**Pull request by mattiacmhpc**
I add into /src/ utilities.c file which contains utilities functions and I add the #include needed as Stefano request

### CODE REVIEW

File: src/utilities.c [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
+  }
 }
 
 /* helper function: apply minimum image convention */
-static double pbc(double x, const double boxby2)
-{
-    while (x >  boxby2) x -= 2.0*boxby2;
-    while (x < -boxby2) x += 2.0*boxby2;
-    return x;
+static double pbc(double x, const double boxby2) {
```
Comment (lleon95) [inline src/utilities.c]: Remove static keyword

File: src/utilities.c [hunk 2]
```diff
@@ -1,46 +1,45 @@
...
- 
+
 /* helper function: zero out an array */
-static void azzero(double *d, const int n)
-{
-    int i;
-    for (i=0; i<n; ++i) {
-        d[i]=0.0;
-    }
+static void azzero(double *d, const int n) {
```
Comment (lleon95) [inline src/utilities.c]: Remove static keyword

File: src/utilities.c [hunk 3]
```diff
@@ -1,46 +1,45 @@
 /* helper function: read a line and then return
    the first string with whitespace stripped off */
-static int get_a_line(FILE *fp, char *buf)
-{
-    char tmp[BLEN], *ptr;
+static int get_a_line(FILE *fp, char *buf) {
```
Comment (lleon95) [inline src/utilities.c]: Remove static keyword

File: src/utilities.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
+A #include<stdio.h>
```
Comment (lleon95) [inline src/utilities.c]: Please, check this typo.

File: src/utilities.c [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+#include <ctype.h>
+#include <string.h>
+
+    /* helper function: read a line and then return
+       the first string with whitespace stripped off */
+    static int get_a_line(FILE *fp, char *buf) {
```
Comment (lleon95) [inline src/utilities.c]: check indentation. Run: `clang-format -i --style=google`

Remove the static keyword

**Review by lleon95 — CHANGES_REQUESTED** [review]

File: src/utilities.c [hunk 6]
```diff
@@ -1,10 +1,10 @@
-A #include<stdio.h>
 #include <ctype.h>
+#include <stdio.h>
 #include <string.h>
```
Comment (lleon95) [inline src/utilities.c]: Project's header is missing. Ask @stefanocampanella  for it

**Review by lleon95 — CHANGES_REQUESTED** [review]

**Review by lleon95 — APPROVED** [review]

File: src/utilities.c [hunk 7]
```diff
@@ -0,0 +1,50 @@
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+#include "../include/ljmod.h"
```
Comment (lleon95) [inline src/utilities.c]: Typo: 

#include "../include/ljmd.h"

**Review by lleon95 — CHANGES_REQUESTED** [review]

File: src/utilities.c [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+/* helper function: read a line and then return
+   the first string with whitespace stripped off */
+int get_a_line(FILE *fp, char *buf) {
```
Comment (lleon95) [inline src/utilities.c]: Delete this function, since it is for INPUT purposes

**Review by lleon95 — CHANGES_REQUESTED** [review]

**Review by stefanocampanella — APPROVED** [review]

**Review by lleon95 — APPROVED** [review]

**Pull request merged**